binary-extensions.json

```diff
@@ -25,10 +29,12 @@
 	"ecelp9600",
 	"eol",
 	"eot",
+	"epub",
+	"exe",
 	"f4v",
 	"fbs",
 	"fh",
-	"flac",
```

Comment:
I meant it is mentioned twice (https://github.com/sindresorhus/binary-extensions/blob/master/binary-extensions.json#L31-L32), so I corrected one of them to point to Adobe's `fla` format instead.